libmscore/cmd.cpp

```diff
@@ -3893,6 +3893,8 @@ void Score::cmd(const QAction* a, EditData& ed)
             { "add-audio",                  [this]{ addAudioTrack();                                            }},
             { "transpose-up",               [this]{ transposeSemitone(1);                                       }},
             { "transpose-down",             [this]{ transposeSemitone(-1);                                      }},
+            { "pitch-up-diatonic-alterations",   [this] { transposeDiatonicAlterations(TransposeDirection::UP); }},
```

Comment:
Here the spaces between `]`and `{` seem unneeded, at least they are not used for the rest of that file